src/commands/commandList/battle/PassiveInterface.js

```diff
@@ -5,118 +5,121 @@
  * For more information, see README.md and LICENSE
  */
 
-const ranks = [0.2, 0.2, 0.2, 0.2, 0.14, 0.05, 0.01];
+const ranks = [0.21, 0.41, 0.61, 0.81, 0.95, 1];
```

Comment:
Because of the other change, we should change this to:

```suggestion
const ranks = [0.2, 0.4, 0.6, 0.8, 0.94, 0.99, 1];
```